blog-service/2024-11-01-observability.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Deprecation Notice - Root Cause Explorer (Observability)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - observability
+  - root cause explorer
+hide_table_of_contents: true
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+As part of our ongoing evaluation of the Sumo Logic service, we have decided to deprecate [Root Cause Explorer](/docs/observability/root-cause-explorer). Root Cause Explorer will continue to function until 30 April 2025.
```

Comment:
```suggestion
As part of our ongoing evaluation of the Sumo Logic service, our product team is deprecating [Root Cause Explorer](/docs/observability/root-cause-explorer), and it will no longer be available as of 30 April 2025.
```